lang/en/texts/contribute-to-open-food-facts-by-playing.html

```diff
@@ -0,0 +1,73 @@
+<!-- no side column -->
+
+<h1>Play and contribute to Open Food Facts!</h1>
+
+<h2 class="subheader">Contributing to Open Food Facts becomes child's play thanks to this game imagined and created by some of our volunteer developers.</h2>
+
+<div class="button_div unselectbuttondiv_front_en">
+	<a href="https://hunger.openfoodfacts.org/" class="button">Play now</button></a>
+<div>
+<p>We thank the many volunteers who have already <a href="https://world-en.openfoodfacts.org/help-complete-products">helped complete products</a> and invite you to discover this game.</p>
+<p><strong>This game allows you to modify products easily, quickly and in a fun way</strong>. Enough to make contributing to an Open Source project addictive.</p>
```

Comment:
"Enough to make contributing to an Open Source project addictive."  --> the "addictive" part is not really positive in my mind. And the sentence without a verb is odd.

Maybe:

"This game allows you to contribute to an open source project that benefits everyone in an easy, quick and fun way!"